artifactory/utils/commandsummary/commandsummary.go

```diff
@@ -176,6 +200,16 @@ func (cs *CommandSummary) getIndexedFileRecursively(dirPath string, isRoot bool)
 	return nestedFilesMap, nil
 }
 
+// Helper function to check if a slice contains a string
+func contains(slice []Index, item string) bool {
```

Comment:
Let's do the following:
* Move this utility function into coreutils/utils.go.
* Rename it to SliceContains.
* Add a unit test for it.